web/.eslintrc.js

```diff
@@ -31,5 +31,11 @@ module.exports = {
     "simple-import-sort/exports": "error",
     "@typescript-eslint/no-unused-vars": "warn",
     "@typescript-eslint/no-empty-function": "warn",
+    'prettier/prettier': [
```

Comment:
Added this rule to help with linter on my end (using Windows) https://stackoverflow.com/a/63654585/16488819

Reply:
If you use Webstorm you won't run into this issue as it lets you specify the end of line character. Regardless let's just leave this as is for those that want to use vs code on windows

Reply:
Probably most editors have a config, but it is good to enforce it. It should probably to be to `'lf'` for unix-style line endings.